Gemfile.lock

```diff
@@ -1,5 +1,5 @@
 GEM
-  remote: https://rubygems.org/
+  remote: http://rubygems.delivery.puppetlabs.net/
```

Comment:
Wouldn't this prevent anyone who was not on our VPN from running any of our rake tasks?